src/features/auth/api.ts

```diff
@@ -8,5 +10,13 @@ type Credentials = {
 
 export async function signin({ email, password }: Credentials): Promise<IUser> {
   const body = { identifier: email, password }
-  return post<IUser>('/users/signin', { body })
+  return post<IUser>('/users/signin', { body, credentials: 'omit' })
+}
+
+export const useCurrentUser = () => {
```

Comment:
Il va falloir qu'on se mette d'accord sur comment écrire nos fonctions^^.
```suggestion
export function useCurrentUser() {
```

Reply:
Vu ensemble, c'est ok. Il faudra voir cela avec d'autres membres de l'équipe si on veut le faire de manière consistente.